<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordConfig</class>
 <widget class="QDialog" name="PasswordConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konfiguracja ochrony hasłem</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/icons/Resources/Icons/lock.png</normaloff>:/icons/Resources/Icons/lock.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/icons/Resources/Icons/lock.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>2</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Konfiguracja ochrony hasłem</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="PasswordConfig_SaveQuit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zapisz i zamknij</string>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/Resources/Icons/save.png</normaloff>:/icons/Resources/Icons/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="PasswordConfig_CancelChanges">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Anuluj zmiany</string>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/Resources/Icons/cancel.png</normaloff>:/icons/Resources/Icons/cancel.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Zapisywanie ustawień</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nowe hasło (max. 25 znaków)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>20</width>
     <height>171</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordConfig_PassNew">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordConfig_PassNew2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nowe hasło:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Powtórz:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordConfig_WarnText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>80</y>
     <width>256</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ochrona hasłem</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="PasswordConfig_Security">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>79</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Nie</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/icons/Resources/Icons/cancel.png</normaloff>:/icons/Resources/Icons/cancel.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tak</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/icons/Resources/Icons/accept.png</normaloff>:/icons/Resources/Icons/accept.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="PasswordConfig_Rules_Run">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>118</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Nie</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/icons/Resources/Icons/cancel.png</normaloff>:/icons/Resources/Icons/cancel.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tak</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/icons/Resources/Icons/accept.png</normaloff>:/icons/Resources/Icons/accept.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>119</y>
     <width>256</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Uruchomienie aplikacji</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>75</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kiedy wymagać podania hasła?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="PasswordConfig_Rules_Settings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>149</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Nie</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/icons/Resources/Icons/cancel.png</normaloff>:/icons/Resources/Icons/cancel.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tak</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/icons/Resources/Icons/accept.png</normaloff>:/icons/Resources/Icons/accept.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>150</y>
     <width>256</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zmiana ustawień</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="PasswordConfig_Rules_Autorun">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>179</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Nie</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/icons/Resources/Icons/cancel.png</normaloff>:/icons/Resources/Icons/cancel.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tak</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/icons/Resources/Icons/accept.png</normaloff>:/icons/Resources/Icons/accept.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>180</y>
     <width>256</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edycja autouruchamiania</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="PasswordConfig_Rules_ShowApp">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>209</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Nie</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/icons/Resources/Icons/cancel.png</normaloff>:/icons/Resources/Icons/cancel.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tak</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/icons/Resources/Icons/accept.png</normaloff>:/icons/Resources/Icons/accept.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>210</y>
     <width>256</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Przywrócenie okna</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordConfig_WarnIcon">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>197</y>
     <width>38</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/icons/Resources/Icons/warning.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="PasswordConfig_WantNew">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>118</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordConfig_NewPassCover">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>136</y>
     <width>347</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/pictures/Resources/Pictures/disabled.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordConfig_SettingsCover">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>111</y>
     <width>349</width>
     <height>126</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/pictures/Resources/Pictures/disabled.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
